cybersecurity/password-analyzer/devashmit/analyzer.py

```
@@ -0,0 +1,132 @@
+"""
+import getpass, math, re, sys
+
+# Top 50 most common passwords (representative sample)
+COMMON_PASSWORDS = {
+    "123456","password","123456789","12345678","12345","1234567","1234567890",
+    "qwerty","abc123","111111","iloveyou","admin","letmein","monkey","1234",
+    "dragon","master","sunshine","princess","welcome","shadow","superman",
+    "michael","football","pass","baseball","soccer","pokemon","batman","ninja",
+    "harley","trustno1","hunter","ranger","buster","thomas","tiger","robert",
+    "charlie","ashley","jessica","jennifer","pepper","joshua","george","andrew",
+    "maggie","666666","hello","whatever","donald","access","696969","batman",
```

Ruff (0.15.15)

[warning] Sets should not contain duplicate item `"batman"`

Remove duplicate item

(B033)